Patch 530 (#551)
aquasecurity/cloud-security-remediation-guides: en/azure/tableservice/table-service-all-access-acl.md

## Detailed Remediation Steps

1. Log in to the Microsoft Azure Management Console.
2. Select the "Search resources, services, and docs" option at the top and search for Storage account. </br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step2.png"/>
3. Select the "Storage account" by clicking on the "Name" link to access the configuration changes. </br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step3.png"/>
4. In the left navigation panel, scroll down and click on "Tables" under "Data storage".</br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step4.png"/>

> 5. On the "tables" page, select the "Table" by clicking on three dot (...) at the right corner to goto "Access policy." </br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step5.png"/>

Reviewer: ```suggestion
5. On the "tables" page, select the "Table" by clicking on three dots (...) at the right corner to go to the "Access policy." </br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step5.png"/>
```

6. In the selected "Table",check the "Permissions" associated with the "Table". If the "Table" allows full Read, Add, Update, Delete ACL permissions then the selected "Table" is not as per the standard configurations.</br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step6.png"/>

> 7. To make changes click on the three dot(...) at the right corner and select the "Edit" option.</br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step7.png"/>

Reviewer: ```suggestion
7. To make changes click on the three dots (...) at the right corner and select the "Edit" option.</br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step7.png"/>
```

8. Uncheck the global read/update/detele policies under the "Permissions" and click on the "OK" button to make the changes.</br> <img src="/resources/azure/tableservice/table-service-all-access-acl/step8.png"/>

> 9. Repeat step number 3 - 8 to ensures "Tables" do not allow full Read, Update, Delete ACL permissions. </br>

Reviewer: ```suggestion
9. Repeat step number 3 - 8 to ensure "Tables" do not allow full Read, Update, Delete ACL permissions. </br>
```

> 9. Repeat steps number 3 - 8 to ensures "Tables" do not allow full Read, Update, Delete ACL permissions. </br>

Reviewer: ```suggestion
9. Repeat step number 3 - 8 to ensures "Tables" do not allow full Read, Update, Delete ACL permissions. </br>
```
